templates/layouts/base.html

```diff
@@ -8,52 +8,27 @@
 {% endif %}
 
 {% set current_path = current_path | default(value="/") %}
-{% if section and section.title %}
-  {% if section.path is starting_with("/learn/book/") %}
-    {% set page_title = "Bevy Book: " ~ section.title %}
-  {% elif section.path is starting_with("/learn/errors/") %}
-    {% set page_title = "Bevy Errors: " ~ section.title %}
-  {% elif section.path is starting_with("/assets") %}
-    {% set page_title = "Bevy Assets" %}
-  {% else %}
-    {% set page_title = section.title %}
-  {% endif %}
-{% elif page and page.title %}
-  {% if page.path is starting_with("/learn/book/") %}
-    {% set page_title = "Bevy Book: " ~ page.title %}
-  {% elif page.path is starting_with("/learn/errors/") %}
-    {% set page_title = "Bevy Errors: " ~ page.title %}
-  {% elif page.path is starting_with("/assets") %}
+
+{% if section_or_page and section_or_page.title %}
+  {% if section_or_page.path is starting_with("/learn/book/") %}
+    {% set page_title = "Bevy Book: " ~ section_or_page.title %}
+  {% elif section_or_page.path is starting_with("/learn/errors/") %}
+    {% set page_title = "Bevy Errors: " ~ section_or_page.title %}
+  {% elif section_or_page.path is starting_with("/assets") %}
     {% set page_title = "Bevy Assets" %}
   {% else %}
-    {% set page_title = page.title %}
+    {% set page_title = section_or_page.title %}
   {% endif %}
 {% else %}
   {% set page_title = "Bevy Engine" %}
 {% endif %}
-{% if section and section.path %}
-  {% set path = "/" ~ section.path %}
-{% elif page and page.path %}
-  {% set path = "/" ~ page.path %}
+
+{% if section_or_page and section_or_page.path %}
+  {% set path = "/" ~ section_or_page.path %}
 {% else %}
   {% set path = "" %}
 {% endif %}
-{% if section %}
-  {% if section.path is starting_with("/learn/book/") %}
-    {% set show_nav_toggle = true %}
-  {% elif section.path is starting_with("/learn/migration-guides/") %}
-    {% set show_nav_toggle = true %}
-  {% elif section.path is starting_with("/learn/errors/") %}
-    {% set show_nav_toggle = true %}
-  {% elif section.path is starting_with("/assets/") %}
-    {% set show_nav_toggle = true %}
-  {% endif %}
-{% elif page %}
-  {% if page.path is starting_with("/learn/book/") or page.path is starting_with("/learn/migration-guides/") or page.path is starting_with("/learn/errors/") or page.path
-    is starting_with("/assets/") %}
-    {% set show_nav_toggle = true %}
-  {% endif %}
-{% endif %}
```

Comment:
`show_nav_toggle` is not used in the codebase, so the whole thing can be removed.